app/config/security.yml

```diff
@@ -8,6 +8,23 @@ security:
     encoders:
         Sylius\Component\User\Model\UserInterface: sha512
     firewalls:
+        admin:
+            switch_user: true
+            context: user
+            pattern: /admin/.*
```

Comment:
yeah fair enough :+1: 

It's just that Wordpress is /admin/ meaning every bot in the world will be looking for it - but trust in the security I guess :)